src/test/java/org/jabref/logic/integrity/IntegrityCheckTest.java

```diff
@@ -194,6 +205,19 @@ void testTitleChecks() {
         assertCorrect(withMode(createContext(StandardField.TITLE, "This is a {Title}"), BibDatabaseMode.BIBLATEX));
         assertCorrect(withMode(createContext(StandardField.TITLE, "{C}urrent {C}hronicle"), BibDatabaseMode.BIBLATEX));
         assertCorrect(withMode(createContext(StandardField.TITLE, "{A Model-Driven Approach for Monitoring {ebBP} BusinessTransactions}"), BibDatabaseMode.BIBLATEX));
+        assertCorrect(withMode(createContext(StandardField.TITLE, "This is a sub title 1: This is a sub title 2"), BibDatabaseMode.BIBLATEX));
```

Comment:
It is good to have tests here. These are more a end-to-end-test. Would you mind to add separate tests to `TitleCheckerTest.java`. In IntelliJ, you can create this file using Ctrl+Shift+T (and pressing Enter in the dialog). It would really help to have the tests for the TitleChecker in a dedicated class, which helps to focus on the implementation. -- This should be very easy to do, but will help future work on the title checker (maybe in one, two years) to ensure that the functionality is still correct.

Reply:
Thank you for your explanation! As suggested, I have added separate tests for titles having delimiters in the new file. Those method names may not be perfect. If there is a better way to rename it, please let me know.